src/alert_rules/prometheus/metrics_alert_rules.yaml

```diff
@@ -11,3 +12,65 @@ groups:
           summary: MySQL Down (instance {{ $labels.instance }})
           description: "MySQL instance is down\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}"
 
+      # 2.1.2
+      # customized: 80% -> 90%
+      - alert: MySQLTooManyConnections(>80%)
+        expr: max_over_time(mysql_global_status_threads_connected[1m]) / mysql_global_variables_max_connections * 100 > 90
+        for: 2m
+        labels:
+          severity: warning
+        annotations:
+          summary: MySQL too many connections (> 90%) (instance {{ $labels.instance }})
```

Comment:
It is 80% in lines 17-18 and 90% here. Typos?

Reply:
oops, sorry, yes typo. corrected in 029f4b826677a277e3f935c317413ecaacf108e4